MediaBrowser.MediaEncoding/Subtitles/VttWriter.cs

```diff
@@ -7,28 +7,39 @@
 
 namespace MediaBrowser.MediaEncoding.Subtitles
 {
+    /// <summary>
+    /// Subtitle writer for the WebVTT format.
+    /// </summary>
     public class VttWriter : ISubtitleWriter
     {
+        /// <inheritdoc />
         public void Write(SubtitleTrackInfo info, Stream stream, CancellationToken cancellationToken)
         {
             using (var writer = new StreamWriter(stream, Encoding.UTF8, 1024, true))
             {
                 writer.WriteLine("WEBVTT");
                 writer.WriteLine(string.Empty);
+                writer.WriteLine("REGION");
+                writer.WriteLine("id:subtitle");
+                writer.WriteLine("width:80%");
+                writer.WriteLine("lines:3");
```

Comment:
Seems like something that shouldn't be hardcoded. Can the client change it?

Reply:
It has to be baked in the WebVTT file, so no.

Ideally, we wouldn't need it, but all browsers seem to render them right against the lower margin of the window if it's not there (And some don't even respect it, like Firefox)

Reply:
Maybe it's outside the scope of the PR, but this should be configurable in some way imo